geode-core/src/main/java/org/apache/geode/internal/memcached/RequestReader.java

```diff
@@ -246,17 +245,18 @@ private void clear(ByteBuffer response) {
     response.clear();
   }
 
-  @MakeImmutable
-  private static byte[] cleanByteArray;
+  @Immutable
+  private static final byte[] cleanByteArray = createCleanByteArray();
 
-  private byte[] getCleanByteArray() {
-    if (cleanByteArray != null) {
-      return cleanByteArray;
-    }
-    cleanByteArray = new byte[RESPONSE_HEADER_LENGTH];
-    for (int i = 0; i < cleanByteArray.length; i++) {
-      cleanByteArray[i] = 0;
+  private static byte[] createCleanByteArray() {
+    byte[] array = new byte[RESPONSE_HEADER_LENGTH];
```

Comment:
maybe just preference, but I did like `cleanByteArray` over just `array` here as a variable name.  A little more explicit in usage/intent.